src/assets.rs

```diff
@@ -583,6 +619,7 @@ impl<'a> AssetEditor<'a> {
             self.asset.mint_info = Some(mint_info);
         }
         self.asset.verified |= other.verified;
+        self.asset.modified_time = Local::now();
```

Comment:
Should we also set `self.asset.created_time = min(self.asset.created_time, other.created_time)`? I.e. when we merge two assets, the created time is whichever was created first